docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -510,15 +510,24 @@ Row level deletion
 Tables using v2 of the Iceberg specification support deletion of individual rows
 by writing position delete files.
 
-Rolling back to a previous snapshot
------------------------------------
+Snapshots
+---------
 
 Iceberg supports a "snapshot" model of data, where table snapshots are
-identified by an snapshot IDs.
+identified by a snapshot ID.
+
+The connector provides a system table containing snapshot information for every
+Iceberg table. Snapshots are identified by ``BIGINT`` snapshot IDs.
+You can find the snapshot IDs for table ``customer_orders`` by running the
+following command::
+
+    SELECT snapshot_id FROM iceberg.testdb."customer_orders$snapshots" ORDER BY committed_at DESC
+
+Rolling back to a previous snapshot
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The connector provides a system snapshots table for each Iceberg table.  Snapshots are
-identified by BIGINT snapshot IDs.  You can find the latest snapshot ID for table
-``customer_orders`` by running the following command::
+You can determine the snapshot IDs for a table in the hidden metadata table ``$snapshots``.
+Use this table to determine the latest snapshot ID with a query similar to the following example::
```

Comment:
Maybe:

Use the \`\`$snapshots\`\` system table to determine the latest snapshot ID like in this example query::